app/views/static_pages/home.html.erb

```diff
@@ -0,0 +1,6 @@
+ <div class="jumbotron mt-4 alert alert-warning offset-md-2 col-md-8 offset-md-2">
+    <h1 class="text-center"><strong>!!Enjoy Learning!!</strong></h1>
+     <p class="text-center">~ Aim to be a master of English ~</p>
+      <img class="d-block mx-auto col-8"
+         src="home_pic.webp" width="500" height="300">
```

Comment:
please change this code to the following:
`<%= image_tag ("home_pic.webp", width: 500, height: 300) %>`